javascript/stack-queue-pseudo/README.md

```diff
@@ -0,0 +1,3 @@
+# Stack-queue-psuedo
+
```

Comment:
try `![whiteboard](Whiteboard.png)` or ![whiteboard](./Whiteboard.png). There is a way in VScode to preview readme. It is a lot like the HTML img tag, where it knows it looking for an image file(jpg, jpeg, png, etc.). Just need to know where that image is located😉.